nix/sources.json

```diff
@@ -1,14 +1,14 @@
 {
     "haskell.nix": {
-        "branch": "master",
+        "branch": "hkm/emcc",
```

Comment:
Fine to use a branch for now - I assume there will be a PR some time?

Does this mean it actually uses emcc to compile the cbits?

Reply:
I'm still not sure I have it right.  Will check with @luite.  Once it is we can merge it and set this back to master.

Reply:
OK, so #108.